www/src/util/helpers.ts

```diff
@@ -0,0 +1,11 @@
+function arrayBufferToBase64(buffer: ArrayBuffer): string {
```

Comment:
Nit: I think this should be a subfile under a `helpers` folder or remove `default` from `export default`

Reply:
fixed